src/components/asset-list/RecyclerAssetList.js

```diff
@@ -746,6 +743,12 @@ class RecyclerAssetList extends Component {
   savingsList = [];
   savingsSumValue = 0;
 
+  scrollInterval = scrollingVelocity => {
```

Comment:
I'm not a fan of the naming here cause it can be confusing,  but I can't think of a better suggestion either  😅